docs/getting-started.md

```diff
@@ -46,3 +60,9 @@ To get a production build, run:
 ```bash
 npm run build:js
 ```
+
+In order bring the entire project up, we will also need to start the php server, run:
```

Comment:
`To run a local instance of WordPress, in a separate terminal:`

Reply:
Done